paimon-web-ui/src/views/playground/components/query/components/tabs/index.tsx

```diff
@@ -15,9 +15,10 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License. */
 
```

Comment:
Why change this format？

Reply:
It's a front-end specification format, with external dependencies uniformly sorted above, local code references sort below, and types below